msg/versioned/ActuatorServos.msg

```diff
@@ -1,11 +1,12 @@
 # Servo control message
+#
+# Normalised output setpoint for up to 8 servos.
+# Published by the vehicle's allocation and consumed by the actuator output drivers.
 
 uint32 MESSAGE_VERSION = 0
 
-uint64 timestamp			# time since system start (microseconds)
-uint64 timestamp_sample	    # the timestamp the data this control response is based on was sampled
+uint64 timestamp # Time since system start [us]
+uint64 timestamp_sample # Sampling timestamp of the data this control response is based on [us]
 
 uint8 NUM_CONTROLS = 8
-float32[8] control # range: [-1, 1], where 1 means maximum positive position,
-                   # -1 maximum negative,
-                   # and NaN maps to disarmed
+float32[8] control # [@range -1, 1] Normalized output. 1 means maximum positive position. -1 maximum negative position (if not supported by the output, <0 maps to NaN). NaN maps to disarmed.
```

Comment:
Note, the bit " (if not supported by the output, <0 maps to NaN)" is **NEW** here - it comes from the ActuatorMotors. Is it also true for servos (if not I will remove)